Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ruby:2.5.1-stretch
+
+ARG DISTRO_NAME=debian
+ARG DISTRO_RELEASE_CODENAME=stretch
+ARG NODE_VERSION=node_8.x
+
+RUN apt-get update -qq \
+  && apt-get install -y \
+    apt-transport-https \
+    build-essential \
+    ca-certificates \
+    curl \
+    git \
+    wget \
+  && wget -O - https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add - \
+  && echo "deb https://deb.nodesource.com/$NODE_VERSION $DISTRO_RELEASE_CODENAME main" >> /etc/apt/sources.list.d/nodesource.list \
+  && echo "deb-src https://deb.nodesource.com/$NODE_VERSION $DISTRO_RELEASE_CODENAME main" >> /etc/apt/sources.list.d/nodesource.list \
+  && curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - \
+  && echo "deb https://dl.yarnpkg.com/$DISTRO_NAME/ stable main" >> /etc/apt/sources.list.d/yarn.list \
+  && apt-get update -qq \
+  && apt-get install -y \
+     nodejs \
```

Comment:
Any thoughts on installing specific versions of these packages?

Reply:
Contributions welome. :trollface:

Reply:
Truthfully, this _is_the goal. I just didn't get that far over lunch yesterday. But I DO think we'd want to pin versions before this is considered merge-able.

Reply:
I think we will need to setup a CI loop that builds this image on the regular if we want to pin it, and make sure that it will continue to work. This will let us catch pulled packages and related issues.